main.py

```diff
@@ -0,0 +1 @@
+print("hello world")
```

Comment:
```suggestion
print("hello world")
```
```suggestion
print("hello world")
```
Look.......
![Screenshot_6](https://user-images.githubusercontent.com/35168657/92313198-9aa61380-efd1-11ea-8471-ef5252bdd40c.png)
.........................